docs/javascript/v2/changelog/release-notes.mdx

```diff
@@ -14,6 +14,15 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hls-player                  | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhls-player.svg)](https://badge.fury.io/js/%40100mslive%2Fhls-player)           |
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+## 2023-08-08
+Released: `@100mslive/hms-video-store@0.10.12`
+
+## Important
+
+Starting from this release, if the beam meeting url passed has sdk version same or above this release,
```

Comment:
"beam meeting url passed" - this is not a user-friendly framing. at the very least, it needs to mention recording or live streaming (beam is an internal term)

Reply:
Hmm, makes sense. How about "if the meeting url passed to start the recording, rtmp, or hls has sdk version same or above this release,"?